docs/user_guide/logging.rst

```diff
@@ -0,0 +1,63 @@
+Logging
+=======
+
+Since version 0.18.0 :program:`rebase-helper` uses standard module-level loggers.
+
+There are also 3 loggers with a special purpose:
+
+=============================== =====================================================================
+Logger name                     Purpose
+=============================== =====================================================================
+:samp:`rebasehelper.traceback`  traceback generated by unhandled exceptions
+:samp:`rebasehelper.summary`    messages forming the rebase summary
+:samp:`rebasehelper.report`     messages forming the textual rebase report (used by text output tool)
```

Comment:
I think `text` should be formatted in some way, to indicate it's a name of a plugin.